### PR TITLE
Change settings layout, fix overflow issue on mobile

### PR DESCRIPTION
**Pull request by P-man2976**
- Changed settings UI layout (Prevents display corruption on mobile)
  - before
image
  - after
image
- Fixed elements overflowing from the screen on mobile
  - before
music-staging holodex net_video_J64qe6P3eO4
  - after
localhost_3000_settings

### CODE REVIEW

**Review by sphinxrave — APPROVED** [review]
ayyyy

File: src/components/settings/LanguageSettings.tsx [hunk 1]
```diff
@@ -6,40 +7,46 @@ import { SettingsSection } from "./SettingsSection";
...
-      credit: "Doubleturtle#3660",
-    },
-    // { value: "it", display: "Italiano", credit: "テオさん#0139" },
-    { value: "tr-TR", display: "Türkçe", credit: "creeperkafasipw#1861" },
-    { value: "vi-VN", display: "Tiếng Việt", credit: "Frincess" },
-    { value: "hu-HU", display: "Magyar", credit: "kuroihikikomori#7216" },
-    // { value: "th", display: "ไทย", credit: "SnowNeko#0282" },
-    // { value: "cimode", display: "Internal Translation Use" },
-  ];
+  const displayLangPrefs = useMemo(
```
Comment (sphinxrave) [inline src/components/settings/LanguageSettings.tsx:10]: you know... instead of using a useMemo, maybe we can just lift this as a const outside... LOL

**Comment by sphinxrave** [comment]
too lazy to wait for that change so gonna merge it.

File: src/components/settings/LanguageSettings.tsx (hunk 1 above)
Comment (P-man2976) [reply src/components/settings/LanguageSettings.tsx:10]: ah yeah lol, I'll fix that in future commits

**Pull request merged**